being-a-good-community-member.md

```diff
@@ -1,50 +1,52 @@
-# Being a Good Citizen of Exercism 
+# Being a Good Citizen of Exercism
 
 We want Exercism to be a great place to spend time, whoever you are.
-To ensure that Exercism is that, we place a strong focus on being good community citizens.
+To ensure that Exercism is that, we place a strong focus on being good community members.
 This is formalised in our [Code of Conduct](https://exercism.io/code-of-conduct) but can be summed up simply in one sentence: **Be kind.**
 
 ## The words we use
 
 There are lots of ways to embrace being kind within Exercism, but the most powerful is to consider how your words affect others.
 
-Words in themselves are meaningless. 
+Words in themselves are meaningless.
 They are vibrations caused by puffs of air coming out of our lungs.
 However, because of **context**, words are powerful.
 
-It's easy to know how you intend your words to _come across_, but it's difficult to know how others _hear your words_.
-What we say is strongly tempered by our experiences in life, our cultural norms, and our primary spoken language.
-At Exercism, we believe it's better to choose words that are **clear** and **unambiguous** across cultures and, where possible, across spoken languages.
+It's easy to know how you intend your words to _come across_, but it's difficult to know how others _perceive your words_.
+What we say is strongly tempered by our experiences in life, our cultural norms, and our primary natural language.
+At Exercism, we believe it's better to choose words that are **clear** and **unambiguous** across cultures and, where possible, across natural languages.
+>>>>>>> 52e9528 (WIP):being-a-good-community-member.md
```

Comment:
There's some left over git thing in here that causes all kinds of weird formatting

Reply:
Thanks.